tests/io/datasets/test_empad.py

```diff
@@ -36,6 +37,16 @@ def default_empad():
     yield ds
 
 
+@pytest.fixture
+def buffered_empad(lt_ctx):
+    buffered = BufferedBackend()
+    return lt_ctx.load(
+        "empad",
+        path=EMPAD_XML,
+        io_backend=buffered,
+    )
+
+
```

Comment:
> Since the default EMPAD fixture doesn't specify the backend and buffered is now the default on Windows, it seems that the MMAP backend will never be used on Windows and the tests will instead run twice with the buffered backend? I just noticed it here, probably applies to other dataset types as well?

Good catch - I'll go through the tests and make sure the default datasets all use the mmap backend. I think this needs to be done for all data set types.

> Do we have to test the default option as well or will it be sufficient to specify the backend in each case?

I'll add a test case that checks automated backend selection depending on system. I think it should be enough to test this with one dataset type, I'll add it for raw.